src/it/scala/org/zalando/spearheads/innkeeper/routes/PatchPathsSpec.scala

```diff
@@ -165,6 +165,23 @@ class PatchPathsSpec extends FunSpec with BeforeAndAfter with Matchers {
         }
       }
 
+      describe("when path patch updates host ids so that a path collision exists") {
+        it("should return the 400 Bad Request status") {
```

Comment:
A really valid point that points to something more concerning. The response doesn't give enough information back pertaining to why it wasn't a successful request. I'll add this change.